mkdocs.yml

```diff
@@ -262,4 +262,13 @@ pages:
   - Source Control: best-practices/source-control.md
   - Continuous Integration: best-practices/continuous-integration.md
   - Keys: best-practices/keys.md
-- Advanced: advanced.md
+- Advanced:
```

Comment:
Can we make this behave similarly to how the `Actions` buttons behaves? Folded by default, and when clicked, it folds up, and shows an overview of all sub-pages basically?

Reply:
I think with our current system this should work without extra effort, right?

Reply:
That's actually quite a bit of extra, hacky work as "Actions" has a sister "_Actions" that holds the action pages and there are multiple places where this is configured to work... But it is possible, sure. Worth the effort?